web/app/mu-plugins/ppj/acf-json/group_5a1c011cdc0e7.json

```diff
@@ -609,6 +636,99 @@
                                     "delay": 0
                                 }
                             ]
+                        },
+                        {
+                            "key": "field_5ae6c1557e526",
+                            "label": "Icon rows",
+                            "name": "icon_rows",
+                            "type": "repeater",
+                            "instructions": "",
+                            "required": 0,
+                            "conditional_logic": [
+                                [
+                                    {
+                                        "field": "field_5a2038941eb03",
+                                        "operator": "==",
+                                        "value": "icon rows"
+                                    }
+                                ]
+                            ],
+                            "wrapper": {
+                                "width": "",
+                                "class": "",
+                                "id": ""
+                            },
+                            "collapsed": "",
+                            "min": 0,
+                            "max": 0,
+                            "layout": "table",
+                            "button_label": "",
```

Comment:
Doesn't this icon rows stuff belong in the `icon-rows` PR? Same with the lead paragraph above – where does that belong?

This may not merge nicely.

Reply:
I suspect I saved an open ACF page after switching branches. I'll clean this up now.

Reply:
This should be working properly now.